### PR TITLE
Fix global flake8 issues

### PR DESCRIPTION
**Pull request by aorenste**
Prior to this `lintrunner --all-files --take FLAKE8` failed.

Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #124771
* #124428

cc @mrshenli @pritamdamania87 @zhaojuanmao @satgera @rohan-varma @gqchen @aazzolini @osalpekar @jiayisuse @H-Huang @kwen2501 @awgu @penguinwu @fegin @XilunWu @wanchaol @fduwjj @wz337 @tianyu-l @wconstab @yf225 @chauhang @d4l3k @voznesenskym @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @peterbell10 @ipiszy @chenyang78 @kadeng @muchulee8 @ColinPeppler @amjames @desertfire

### CODE REVIEW

**Review by Skylion007 — APPROVED** [review]
Next PR will add the file to pyproject.toml for flake8+ruff?

**Comment by aorenste** [comment]
> Next PR will add the file to pyproject.toml for flake8+ruff?

I'm not sure what you mean by this...

**Comment by Skylion007** [comment]
> > Next PR will add the file to pyproject.toml for flake8+ruff?
> 
> I'm not sure what you mean by this...

Oh sorry, I meant make sure the files are covered by the lintrunner.toml configuration so they are checked automatically from now on by flake8 and mypy etc.

**Comment by aorenste** [comment]
Relanding - was reverted because of an unrelated error in #123735

@pytorchbot merge

File: test/test_jit.py [hunk 1]
```diff
@@ -15679,7 +15679,7 @@ def fn(string):
     def test_unicode_comments(self):
         @torch.jit.script
         def test(self, a):
-            # 🤷🤷🤷🤷
+            # shrug
```
Comment (malfet) [inline test/test_jit.py:15682]: Well, this BE change actually renders test useless, as it supposed to test unicode comments... :/

Comment (aorenste) [reply test/test_jit.py:15682]: Sure - It seems that we should either tell flake8 to allow unicode or we should not have code with unicode in it.

**Pull request closed without merging**